YoCode/Output/WebWriter.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace YoCode
+{
+    public class WebWriter : IPrint
+    {
+        const string TEMPLATE_FILE = @"C:\Users\ukekar\source\repos\YoCode\YoCode\Output\HTMLTemplate.html";
+        const string OUTPUT_FILE = @"C:\Users\ukekar\source\repos\YoCode\webReport\output.html";
```

Comment:
We should definitely not have absolute paths here. The template should probably be a resource in the project, like all our string messages are now. The output file should probably default to going in whatever the current working directory is. Also, it should probably have a more descriptive name, like "YoCodeReport.html" - "output.html" could easily be confused with something else.